cylc/rose/utilities.py

```diff
@@ -523,3 +530,81 @@ def paths_to_pathlib(paths):
         else None
         for path in paths
     ]
+
+
+def override_this_variable(node, section, variable):
+    """Variable exists in this section of the config and should be replaced
+    because it is a standard variable.
+
+    Examples:
+        Setup:
+        >>> from metomi.rose.config import ConfigNode
+        >>> from cylc.rose.utilities import (
+        ... ROSE_ORIG_HOST_INSTALLED_OVERRIDE_STRING as rohios
+        ... )
+        >>> node = ConfigNode()
+        >>> node = node.set(['env'])
+
+        1. Variable not in node[section]:
+        >>> override_this_variable(node, 'env', 'foo')
+        False
+
+        2. Variable is not ROSE_ORIG_HOST:
+        >>> node = node.set(['env', 'ROSE_VERSION'], '123.456')
+        >>> override_this_variable(node, 'env', 'ROSE_VERSION')
+        True
+
+        3. Variable is ROSE_ORIG_HOST and override string unset:
+        >>> node = node.set(['env', 'ROSE_ORIG_HOST'], '123.456.789.10')
+        >>> override_this_variable(node, 'env', 'ROSE_ORIG_HOST')
+        True
+
+        4. Variable is ROSE_ORIG_HOST and override string set:
+        >>> node['env']['ROSE_ORIG_HOST'].comments = [rohios]
+        >>> override_this_variable(node, 'env', 'ROSE_ORIG_HOST')
+        False
+    """
+    if variable not in node[section]:
+        return False
+    elif (
+        variable != 'ROSE_ORIG_HOST'
+        or (
+            variable in node[section]
+            and ROSE_ORIG_HOST_INSTALLED_OVERRIDE_STRING not in
```

Comment:
Think that's extraneous as we're already inside the `if variable not in node[section]: ... elif:`
```suggestion
            ROSE_ORIG_HOST_INSTALLED_OVERRIDE_STRING not in
```